modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTypeSettings.java

```diff
@@ -98,6 +99,11 @@
 public interface NumericDDMFormFieldTypeSettings
 	extends DefaultDDMFormFieldTypeSettings {
 
+	@DDMFormField(
+		dataType = "numeric", label = "%character-options", type = "help_text"
```

Comment:
@mateusgomes01 Why did you set the `dataType` to numeric if the `characterOptions` field is not numeric? Do you need this in the front-end code? If not, you can remove this property.

Reply:
Ok!